packages/docs/PropTables/ActionBarPropTable.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { NextLink, Prop, PropTable, PropTableWrapper } from '../components';
+
+const actionBarProps: Prop[] = [
+  {
+    name: 'actions',
+    types: (
+      <>
+        {'Array<'}
+        <NextLink
+          href={{ hash: 'action-button-prop-table', query: { props: 'action-button-props' } }}
+        >
+          ActionButton
+        </NextLink>
+        {'>'}
```

Comment:
For a singular type array, let's use the following:
```suggestion
        <NextLink
          href={{ hash: 'action-button-prop-table', query: { props: 'action-button-props' } }}
        >
          ActionButton
        </NextLink>
        []
```